### DISCUSSION TITLE
Can a used Slingbox M500/M300 etc be setup with Dazoo server after SB server shutdown?

### DISCUSSION THREAD

**Discussion by robosmith** [GerryDazoo/Slinger#247] [Q&A]
Assuming factory reset with default password, can later (used) Slingbox models work with the Dazoo server when set up after the SB server shutdown?

**Comment by AndyJStubbs** [comment]
If you mean Gerry's slingbox_server then yes they should work assuming you know your current admin password or have done a factory reset and are using the default 'admin' password after shutdown.

**Comment by GerryDazoo** [comment]
We don't know how to download the latest firmware or configure a video input Yet, so....

**Reply by AndyJStubbs** [reply]
That's an interesting point - do you actually need to configure the input? I mean you can select which input is used via your slingbox server so what does the slingbox itself need to know?

Firmware I guess now it doesn't need to talk to sling servers may be a moot point - It doesn't have to check how many adverts to dish up etc or log you in to sling so I can't imagine too much going wrong on that front (unless the box has ssl certs that need updating etc which i can't see why it would).

**Reply by GerryDazoo** [reply]
Yes, it you want to use the remote control.
No SSL certs.
It was mainly the SlinboxPLayer that needed to talk to the servers. In normal operation the SB would occasionally check for firmware updates and send a message to myip.sling.com to let the cloud know where it was.

**Reply by robosmith** [reply]
IME, at least with cell phones, factory reset does not revert to original factory firmware. If that's also true with Slingboxes, and it was in continual use, the latest firmware would already be installed even after a factory reset. In cell phones, factory reset only clears out the soft settings applied by the user.

**Reply by GerryDazoo** [reply]
Different slingbox models behave differently based on their hardware capabilities. In my testing it looked to me like the Solo reset blasted the box back to a state where it needed a firmware download for normal operation. The 500 behaved differently, like your phone. After the factory reset it was still running the latest firmware but with all the settings wiped out.
